test/stronger_parameters/controller_support/permitted_parameters_test.rb

```diff
@@ -2,7 +2,7 @@
 require_relative '../../test_helper'
 require 'stronger_parameters/controller_support/permitted_parameters'
 
-SingleCov.covered! uncovered: (ActiveSupport::VERSION::MAJOR < 5 ? 3 : 2) # uncovered branches for rails version check
+SingleCov.covered! uncovered: 2
```

Comment:
ideally remove `uncovered: 2` here and leave magic `# uncovered` comments in the lines that are uncovered

Reply:
I don't know which lines are uncovered, so I'm keeping it as it is for now and will play with it after tests pass.